codegen/module_system.go

```diff
@@ -690,6 +691,16 @@ func (g *EndpointGenerator) generateEndpointFile(
 			strings.Title(method.Name) + "Endpoint"
 	}
 
+	// TODO: http client needs to support multiple thrift services
+	var clientMethodName string
+	if method.DownstreamMethod != nil {
```

Comment:
If it is nil, `clientMethodName` will be `""` after this if. Shouldn't it be an error to render template with `""` `ClientMethodName`?